build.gradle

```diff
@@ -90,44 +91,44 @@ configurations.all
 
 dependencies
 {
-  compile 'commons-lang:commons-lang:2.6'
-  compile 'commons-io:commons-io:2.5'
-  compile 'net.sf.trove4j:trove4j:3.0.3'
-  compile 'args4j:args4j:2.0.29'
-  compile 'com.apple:AppleJavaExtensions:1.4'
-  compile 'com.miglayout:miglayout:3.7.4'
+  implementation 'commons-lang:commons-lang:2.6'
+  implementation 'commons-io:commons-io:2.5'
+  implementation 'net.sf.trove4j:trove4j:3.0.3'
+  implementation 'args4j:args4j:2.0.29'
+  implementation 'com.apple:AppleJavaExtensions:1.4'
+  implementation 'com.miglayout:miglayout:3.7.4'
 	
   //ClearCL:
-  compile ('net.clearvolume:clearcl:0.5.2') {transitive= true; changing= true}
+  implementation ('net.clearvolume:clearcl:0.5.2') {transitive= true; changing= true}
```

Comment:
ClearCL and ClearAudio could potentially be made into `api` dependencies, instead of `implementation`. `implementation` means they're kinda private, while `api` will expose them.

Reply:
Also, both `transitive` and `changing` options should be removed. `transitive` is true by default, and changing is only relevant for SNAPSHOTs.

Reply:
I changed them into api dependencies (which prompted addition of `apply plugin java-library`) and removed the `transitive `and `changing `options.